{"links": {"self": {"href": "data/repositories/hoffmanlab/genomedata/pullrequests/27/comments/66528653.json"}, "code": {"href": "https://api.bitbucket.org/2.0/repositories/hoffmanlab/genomedata/diff/mjmm13/genomedata-py2_3-test:dc73c6cc0b14..46366c4b8f1e?path=test%2Ftest_genomedata.py"}, "html": {"href": "#!/hoffmanlab/genomedata/pull-requests/27/_/diff#comment-66528653"}}, "parent": {"id": 66393284, "links": {"self": {"href": "data/repositories/hoffmanlab/genomedata/pullrequests/27/comments/66393284.json"}, "html": {"href": "#!/hoffmanlab/genomedata/pull-requests/27/_/diff#comment-66393284"}}}, "deleted": false, "pullrequest": {"type": "pullrequest", "id": 27, "links": {"self": {"href": "data/repositories/hoffmanlab/genomedata/pullrequests/27.json"}, "html": {"href": "#!/hoffmanlab/genomedata/pull-requests/27"}}, "title": "Pipeline Update and Python 3 Compatibility"}, "content": {"raw": "A warning will appear because we close the genome without calling close data in the context manager. The same issue was occurring in Python 2 however, unittest was apparently masking it. The same thing is being done where the warnings would also appear in Python 2 in a few places, such as in load\\_seq at line 243, albeit without a context manager. ", "markup": "markdown", "html": "<p>A warning will appear because we close the genome without calling close data in the context manager. The same issue was occurring in Python 2 however, unittest was apparently masking it. The same thing is being done where the warnings would also appear in Python 2 in a few places, such as in load_seq at line 243, albeit without a context manager. </p>", "type": "rendered"}, "created_on": "2018-06-07T19:58:43.572166+00:00", "user": {"display_name": "Matthew McNeil", "uuid": "{018cd21a-1658-4202-9b1e-99dabc3e14d0}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B018cd21a-1658-4202-9b1e-99dabc3e14d0%7D"}, "html": {"href": "https://bitbucket.org/%7B018cd21a-1658-4202-9b1e-99dabc3e14d0%7D/"}, "avatar": {"href": "data/secure.gravatar.com/avatar/d7e7604c308bd957d77e1762b5caeddcd=httpsavatar-management--avatars.us-west-2.prod.public.atl-paas.netinitialsMM-5.png"}}, "nickname": "mjmm13", "type": "user", "account_id": "5ac790531b5dd37ea53a123d"}, "inline": {"to": null, "from": null, "outdated": true, "path": "test/test_genomedata.py"}, "updated_on": "2018-06-07T19:58:43.580995+00:00", "type": "pullrequest_comment", "id": 66528653}